roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -93,9 +137,20 @@
   register: existing_metrics_rc
   changed_when: no
 
+- command: >
+    {{openshift_client_binary}}
+    --config={{mktemp.stdout}}/admin.kubeconfig
+    get deployment
+    -l k8s-app=metrics-server
+    -o name
+    -n {{openshift_metrics_server_project}}
+  register: existing_metrics_server_rc
+  changed_when: no
+
 - name: Scaling down cluster to recognize changes
   include_tasks: stop_metrics.yaml
-  when: existing_metrics_rc.stdout_lines | length > 0
+  when: (existing_metrics_rc.stdout_lines | length > 0) or
+        (existing_metrics_server_rc.stdout_lines | length > 0)
```

Comment:
o_O what is the purpose of this?

Reply:
namely, why are we stop metrics to "recognize changes"?  I think this is legacy hawkular cruft...